src/background.js

```diff
@@ -72,24 +76,24 @@ const stateManager = {
 
   async rememberTRRMode() {
     let curMode = await browser.experiments.preferences.getIntPref(TRR_MODE_PREF, 0);
-    log("Saving current trr mode:", curMode);
+    await log("Saving current trr mode:", curMode);
```

Comment:
Since this doesn't need to be async anymore, this `await`, and all the others, can be removed.